client/src/pages/Profile/ProfileEdit/ProfileEditForm/ProfileEditForm.tsx

```diff
@@ -108,12 +103,19 @@ export default function ProfileEditForm(): JSX.Element {
                 address,
                 description,
               };
-
-              setTimeout(() => {
+              setTimeout(async () => {
                 if (!hasProfile) {
-                  axios.post('/profile/', modifiedValues);
+                  await axios.post('/profile/', modifiedValues);
+                  updateSnackBarMessage('Profile created!');
                 } else {
-                  axios.patch('/profile/', modifiedValues);
+                  await axios
+                    .patch('/profile/', modifiedValues)
+                    .catch((error) => {
+                      updateSnackBarMessage(error);
+                    })
+                    .then((response) => {
+                      updateSnackBarMessage('success!');
+                    });
```

Comment:
It's better to move these into the API layer (e.g. functions in `helpers/APICalls/`) 👍🏼